### PR TITLE
Display importer warnings in datasets

### LINKED ISSUE #89: Display a message notifying users when the application doesn’t have routing information
We can display an alert when a user is importing sites outside the area where we have routing information.

We could display something like "warning: there are some sites outside the area where we have routing information"

### PR DESCRIPTION
**Pull request by spalladino**
Fixes #89 

screen shot 2016-08-26 at 7 08 18 pm

### CODE REVIEW

File: src/planwise/component/importer.clj [hunk 1]
```diff
@@ -89,18 +89,23 @@
...
 (defn process-facilities
   [facilities facility-ids]
-  (doseq [id facility-ids]
-    (info "Processing facility" id)
-    (facilities/preprocess-isochrones facilities id)))
+  (map
+    (fn [id]
+      (info "Processing facility" id)
+      (facilities/preprocess-isochrones facilities id))
+    facility-ids))
```
Comment (ggiraldez) [inline src/planwise/component/importer.clj]: `map` returns a lazy sequence which may not be realised in the right moment, or the correct thread for that matter. Use `doall`, `mapv` or `(into [] (map ..))` to force immediate computation.

File: src/planwise/model/import_job.clj [hunk 2]
```diff
@@ -142,15 +143,26 @@
         (assoc :result :update-projects-failed
                :error-info error))))
 
+(defn- update-facilities-counts [job event]
+  (let [[_ _ results] event
+        no-road-network (filter #{:no-road-network} results)
+        outside-regions (filter #{:outside-regions} results)]
+    (-> job
+      (update :facilities-without-road-network-count + (count no-road-network))
+      (update :facilities-outside-regions-count + (count outside-regions)))))
```
Comment (ggiraldez) [inline src/planwise/model/import_job.clj]: Meet http://clojuredocs.org/clojure.core/frequencies :)

Comment (spalladino) [inline src/planwise/model/import_job.clj]: Fixed in 35598b6f50a5ae4d1229d19bbf3f9086c5b92cea

File: src/planwise/component/importer.clj (hunk 1 above)
Comment (spalladino) [inline src/planwise/component/importer.clj]: Fixed in b804d2bf53ec3a24300f80b5e62e5170eeb49c8b

**Pull request merged**